samples/basic-order-processing/src/client/index.ts

```diff
@@ -0,0 +1,102 @@
+import { createClient } from "@amqp-contract/client";
```

Comment:
client application should be a dedicated application in samples

we want to split contract, schema and worker in order to simulate real life with several application communicating together